userconfigs_generator/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"strings"
+
+	"github.com/aiven/aiven-go-client/tools/exp/dist"
+)
+
+const destination = "./api/v1alpha1/userconfigs"
+
+func main() {
+	var serviceList string
+	flag.StringVar(&serviceList, "services", "", "Comma separated service list of names to generate for")
```

Comment:
could we please use https://github.com/spf13/cobra to manage CLIness of this tool, as this is the preferred way of introducing CLI tools within PRs to Go projects of Aiven organization?

`cobra` is a drop-in replacement for built-in `flag` package, and it shouldn't be too hard to switch to it here, please see https://github.com/aiven/aiven-go-client/blob/master/tools/exp/cmd/root.go for example usage

Reply:
Cobra is an overkill for a just one flag and an unnecessary dependancy with indirect dependencies.

Reply:
well, that was exactly what happened to my PR before as well, and I believe there might be a specific reason why we would want to use Cobra

perhaps @ivan-savciuc could share some additional info on it, since they asked me previously to add Cobra when I were introducing a simple CLI tool with one flag

Reply:
I do not see a point of using Cobra framework here.